### PR TITLE
Feat/ccn3 2686/from address field

### PR DESCRIPTION
**Pull request by trichmond-IS**
## Description

[Goes here]

## Todo's before PR

- [ ] Branch has been rebased against develop
- [ ] Unit tests passed (`make test`)
- [ ] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [ ] PR description describes the overall goals of PR commits
- [ ] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [ ] Specified reviewers (even if it's yourself)
- [ ] Specified assignees (those who'll merge)
- [ ] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/tests/test_declaration_and_payment/test_routing.py [hunk 1]
```diff
@@ -93,11 +93,6 @@ def test_doesnt_display_private_information_for_adults_in_the_home(self):
             moved_in_month=2,
             moved_in_year=1980)
 
-        # applicant = models.ApplicantPersonalDetails.objects.get(application_id=self.application)
```
Comment (jenniferbrown94) [inline application/tests/test_declaration_and_payment/test_routing.py]: can this be deleted?

File: application/tests/test_declaration_and_payment/test_routing.py [hunk 2]
```diff
@@ -93,11 +93,6 @@ def test_doesnt_display_private_information_for_adults_in_the_home(self):
...
-        # applicant.update(moved_in_day=2,
```
Comment (jenniferbrown94) [inline application/tests/test_declaration_and_payment/test_routing.py]: Can this be deleted?

**Comment by trichmond-IS** [comment]
rerun jenkins build

File: application/tests/test_declaration_and_payment/test_routing.py (hunk 1 above)
Comment (trichmond-IS) [reply application/tests/test_declaration_and_payment/test_routing.py]: yes, as test is only related to info for people in the home

**Pull request merged**